docs/azuredisk-issues.md

```diff
@@ -319,3 +323,44 @@ This is a common k8s issue, other cloud provider would also has this issue. Ther
 
 **Work around**:
 delete pod first and then delete azure disk pvc after a few minutes
+
+### 12. create azure disk PVC failed due to account creation failure
+ > pls note this issue only happens on **unmanaged** k8s cluster
```

Comment:
pls -> please

Reply:
seems we have met such typos for many times, it's better to add a spell checking in our CI

Reply:
fixed